.vscode/extensions.json

```diff
@@ -1,6 +1,11 @@
 {
     "recommendations": [
         "ms-dotnettools.csharp",
-        "editorconfig.editorconfig"
+        "editorconfig.editorconfig",
+        "ertanic.robust-lsp",
+        "GitHub.vscode-pull-request-github",
+        "eamodio.gitlens",
+        "macabeus.vscode-fluent",
+        "redhat.vscode-yaml"
```

Comment:
Think overall I'd say the PR one is essential but the rest are take-or-leave for most devs.